sympy/stats/rv.py

```diff
@@ -1062,7 +1065,9 @@ def sample(expr, condition=None, size=(), library='scipy',
         Choose any of the available options to sample from as string,
         by default is 'scipy'
     numsamples : int
-        Number of samples, each with size as ``size``
+        Number of samples, each with size as ``size``.
+        numsamples has been deprecated since v1.9.
+        see https://github.com/sympy/sympy/pull/21590#discussion_r649227690.
```

Comment:
You don't need to reference the github comment here. I would say something like
```
numsamples: int
    Number of samples, each with size as ``size``. The ``numsamples`` parameter is
    deprecated and is only provided for compatibility with v1.8. Use a list comprehension
    or an additional dimension in ``size`` instead.
```

Reply:
Changed.